docs/source/getstarted.rst

```diff
@@ -60,74 +61,202 @@ Let's test your new setup by running a very simple program. This program instruc
+
+4. Now type in the following command to have Cozmo drive straight:
+
+.. code-block:: python
+ :lineno-start: 5
+
+   robot.drive_straight(distance_mm(150), speed_mmps(50)).wait_for_completed()
+..
+
+  a. The ``drive_straight`` function creates an action on Cozmo that drives him in a straight line.  It needs a distance and a speed as input.
+  b. ``distance_mm(150)`` is the distance to drive (150 millimeters)
+  c. ``speed_mmps(50)`` is the speed to drive at (50 millimeters per second)
+  d. ``wait_for_completed()`` instructs the program to wait until the drive_straight action has finished before continuing. Recall that we used the same command with say_text in the Hello World and Night Night examples. Say_text and drive_straight are two of cozmo’s “actions” - we used ``wait_for_completed`` to make sure that Cozmo executes these actions one at a time. Later on we will explore how Cozmo can do multiple things at the same time.
+
+5. Now type in the following command to have Cozmo turn around:
+
+.. code-block:: python
+ :lineno-start: 6
+
+ robot.turn_in_place(degrees(180)).wait_for_completed()
+..
+
+  a. ``robot.turn_in_place`` directs Cozmo to turn in place.
+  b. ``(degrees(180))`` sets how far he turns in relation to where he is. Cozmo's initial position is assumed to be 0 degrees; he will turn 180 degrees, or directly around. To make Cozmo turn clockwise, enter a negative number. For example, entering -90 makes Cozmo turn 90 degrees to the right.
+  c. ``wait_for_completed()`` makes sure Cozmo completes his turn before performing his next action.
+
+6. Now copy and paste lines 5 and 6 so that the completed program should look like this:
+
+.. code-block:: python
+ :linenos:
+
+ import cozmo
+ from cozmo.util import distance_mm, speed_mmps
+
+ def cozmo_program(robot: cozmo.robot.Robot):
+  robot.drive_straight(distance_mm(150), speed_mmps(50)).wait_for_completed()
+  robot.turn_in_place(degrees(180)).wait_for_completed()
+  robot.drive_straight(distance_mm(150), speed_mmps(50)).wait_for_completed()
+  robot.turn_in_place(degrees(180)).wait_for_completed()
+
+ cozmo.run_program(cozmo_program)
+..
+
+Save this file as there_and_back.py.
+
+Now you can run your program:
+
+    a. For macOS and Linux systems, type the following into the same Terminal window and press **Enter**::
+
+      ./there_and_back.py
+
+    b. For Windows systems, type the following into the same Command Prompt window and press **Enter**::
+
+      py there_and_back.py
+
+
+Now that you have written your own Cozmo program, take a look at the rest of the Cozmo SDK package at some of our many other example programs to get more ideas. 
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Challenges
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some challenges to try once you have explored the examples (these are hard - try your best!):
```

Comment:
I like the idea of the challenges section, I think we might want to think a bit deeper on the specific challenges and wording.